cluster-autoscaler/integration/framework.go

```diff
@@ -206,3 +244,121 @@ func (driver *Driver) scaleWorkload(workloadName string, replicas int32) error {
 	}
 	return nil
 }
+
+func (driver *Driver) getFirstNode() (*v1.Node, error) {
+	nodeList, err := driver.targetCluster.Clientset.CoreV1().Nodes().List(context.Background(), metav1.ListOptions{})
+	if err != nil {
+		return &v1.Node{}, err
+	}
+	sort.Slice(nodeList.Items, func(i, j int) bool {
+		return nodeList.Items[i].ObjectMeta.CreationTimestamp.Before(&nodeList.Items[j].ObjectMeta.CreationTimestamp)
+	})
+	firstNode := nodeList.Items[0]
+	return &firstNode, nil
+}
+
+func (driver *Driver) getLatestNode() (*v1.Node, error) {
+	nodeList, err := driver.targetCluster.Clientset.CoreV1().Nodes().List(context.Background(), metav1.ListOptions{})
+	if err != nil {
+		return &v1.Node{}, err
+	}
+	sort.Slice(nodeList.Items, func(i, j int) bool {
+		return nodeList.Items[i].ObjectMeta.CreationTimestamp.Before(&nodeList.Items[j].ObjectMeta.CreationTimestamp)
+	})
+	if len(nodeList.Items) < 1 {
+		err = fmt.Errorf("no nodes found")
+	}
```

Comment:
add this just after listing the nodes and before sorting.

Reply:
done